.github/workflows/docker.yml

```diff
@@ -0,0 +1,290 @@
+on:
+  push:
+    branches: [master, staging, trying]
+    tags:
+      - "v*.*.*"
+
+name: Docker Build
+
+jobs:
+  docker-build:
+    needs: [docker-builds]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Docker builds done
+        run: exit 0
+  docker-builds:
+    name: Docker Build
+    runs-on: ${{ matrix.image }}
```

Comment:
this should work, but we're getting:

```
Docker Build : .github#L1
Error when evaluating 'runs-on' for job 'docker-builds'. .github/workflows/docker.yml (Line: 18, Col: 14): Unexpected value ''
```

matrix.image should be `ubuntu-latest` always, unless the include overrides it

Reply:
https://github.com/rust-embedded/cross/actions/runs/1478440457

Reply:
it doesn't work like I thought it did :D